Makefile

```diff
@@ -51,6 +52,10 @@ verify: $(VERIFY)
 etcd:
 	$(MAKE) -C tests/e2e/ etcd &
 
+.PHONY: etcd-travis
+etcd-travis:
+	$(MAKE) -C tests/e2e/ etcd &
```

Comment:
why? this is code duplication

Reply:
as we discussed today, I want to separate travis directives from other make directives

Reply:
ok, then I misunderstood you ... and now that I understand that you are simply duplicate all entries then I object to this ... may understanding was that in cases that the travis target differs from the standard target then you create a travis specific, the example you gave me was:
- travis - running in background
- during development - running in foreground

the duplication the never beneficial as it means that you can never stabilize the project (you fix one place but the other is still broken).

so now that I understand what you are doing, please explain what what is the benefit of duplication which out weighs the obvious downside.

Reply:
Will be a separate PR